hazelcast/src/test/java/com/hazelcast/internal/management/dto/MapConfigDTOTest.java

```diff
@@ -0,0 +1,139 @@
+import com.hazelcast.test.HazelcastParallelParametersRunnerFactory;
+import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Collection;
+
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(Parameterized.class)
+@Parameterized.UseParametersRunnerFactory(HazelcastParallelParametersRunnerFactory.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
+public class MapConfigDTOTest {
+
+    @Parameterized.Parameter(0)
+    public MapConfig mapConfig;
+
+    @Parameterized.Parameters(name = "{0}")
+    public static Collection<Object[]> parameters() {
+        return asList(new Object[][]{
+                {defaultMapConfig()},
+                {fullMapConfig()},
+        });
+    }
+
+    private static final String MAP_NAME = "map-name";
+    private static final ConfigCompatibilityChecker.MapConfigChecker MAP_CONFIG_CHECKER
+            = new ConfigCompatibilityChecker.MapConfigChecker();
+
+    static MapConfig defaultMapConfig() {
+        return new MapConfig(MAP_NAME);
+    }
+
+    static MapConfig fullMapConfig() {
+        MapConfig mapConfig = new MapConfig(MAP_NAME);
+        mapConfig.setBackupCount(2)
+                .setAsyncBackupCount(3)
+                .setCacheDeserializedValues(CacheDeserializedValues.ALWAYS)
+                .setEvictionPolicy(EvictionPolicy.RANDOM)
+                .setInMemoryFormat(InMemoryFormat.NATIVE)
+                .setSplitBrainProtectionName("triple");
+
+        NearCacheConfig nearCacheConfig = new NearCacheConfig();
+        nearCacheConfig.setLocalUpdatePolicy(NearCacheConfig.LocalUpdatePolicy.CACHE_ON_UPDATE);
+        nearCacheConfig.getPreloaderConfig().setEnabled(false).setStoreIntervalSeconds(11)
+                .setStoreInitialDelaySeconds(33).setDirectory("storeHere");
+        mapConfig.setNearCacheConfig(nearCacheConfig);
+
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setInitialLoadMode(MapStoreConfig.InitialLoadMode.EAGER);
+        mapStoreConfig.setClassName("map-store-impl-class-name");
+        mapConfig.setMapStoreConfig(mapStoreConfig);
+
+        MergePolicyConfig mergePolicyConfig = new MergePolicyConfig();
+        mergePolicyConfig.setBatchSize(111);
+        mergePolicyConfig.setPolicy(PassThroughMergePolicy.class.getName());
+
+        return mapConfig;
+    }
+
+    @Test
+    public void cloned_config_with_json_serialization_equals_given_config() {
+        MapConfig clonedConfig = cloneWithJsonSerialization(mapConfig);
+        assertTrue("Expected: " + mapConfig + ", got:" + clonedConfig,
+                MAP_CONFIG_CHECKER.check(mapConfig, clonedConfig));
+    }
+
+    @Test
+    public void cloned_config_with_json_serialization_equals_default_config() {
```

Comment:
seems this test can be deleted, it is same test with `cloned_config_with_json_serialization_equals_given_config`.